cleanrl_utils/huggingface.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import os
+import sys
+from pathlib import Path
+from pprint import pformat
+from typing import List
+
+import numpy as np
+
+HUGGINGFACE_VIDEO_PREVIEW_FILE_NAME = "replay.mp4"
+HUGGINGFACE_README_FILE_NAME = "README.md"
+
+
+def push_to_hub(
+    args: argparse.Namespace,
+    episodic_returns: List,
+    repo_id: str,
+    algo_name: str,
+    folder_path: str,
+    video_folder_path: str = "",
+    revision: str = "main",
+    create_pr: bool = False,
+    private: bool = False,
+):
+    # Step 1: lazy import and create / read a huggingface repo
+    from huggingface_hub import CommitOperationAdd, CommitOperationDelete, HfApi
+    from huggingface_hub.repocard import metadata_eval_result, metadata_save
+
+    api = HfApi()
+    repo_url = api.create_repo(
+        repo_id=repo_id,
+        exist_ok=True,
+        private=private,
+    )
+    # parse the default entity
+    entity, repo = repo_url.split("/")[-2:]
+    repo_id = f"{entity}/{repo}"
```

Comment:
I'm really sorry this is not easier at the moment :confused: 
I made a PR to fix this but it will only be available after next release of `huggingface_hub`. I can open another PR on "cleanrl" once it's release if you want (and you can keep this manual parsing for now, that's fine).

Reply:
Thanks for the comment! Another PR once it's released would be good :)

Reply:
Will do, it's not urgent anyway :)